db/data/20220729144112_delete_empty_degrees.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class DeleteEmptyDegrees < ActiveRecord::Migration[6.1]
+  def up
+    degrees = Degree.where(
```

Comment:
@claire-hughez from a product and data perspective, are you happy for me to delete degrees with the following criteria? I know you and Andrew had a look into them previously. This removes approx 7475 degrees.